lib/checkcondition.cpp

```diff
@@ -1936,8 +1936,7 @@ void CheckCondition::checkCompareValueOutOfTypeRange()
     if (!mSettings->severity.isEnabled(Severity::style) && !mSettings->isPremiumEnabled("compareValueOutOfTypeRangeError"))
         return;
 
-    if (mSettings->platform.type == Platform::Type::Native ||
```

Comment:
The check never ran on Native since https://github.com/danmar/cppcheck/commit/cbb07b6247ed9b6c85766b12011ffc783f5b7626